app/src/lib.rs

```diff
@@ -180,6 +180,12 @@ pub fn setup_logging(log_file_path: Option<PathBuf>, level: LevelFilter) -> Resu
     }
 
     log::set_max_level(level);
+
+    // Dump panic info and backtrace to logger.
+    log_panics::Config::new()
+        .backtrace_mode(log_panics::BacktraceMode::Resolved)
+        .install_panic_hook();
```

Comment:
Looks like this is equivalent to simply calling
```
log_panics::init()
```

ref: https://docs.rs/log-panics/latest/log_panics/

Reply:
I am afraid they are not the same. I tried `log_panics::init()` ending up with no panic backtrace and resolved function name?